### ISSUE TITLE
[Screen Reader-Header Control-Projects]: Proper role is not defined for the 'New Project', 'Import File' and 'Untitled' elements showing under Project dialog box.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2520]
**Pre-Requisite:**  Turn on Narrator(Win + Ctrl + Enter)

**User Experience:**
Users who depends on Screen Reader will get confuse if Name /Role/Control type property is not defined for any control of the web page.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool used:-IE Developer (F12)

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Projects control lying in the header section on the page and select it.
4. Navigate to various controls lying on the page opened.
5. Verify that proper role is defined for all the elements showing on Project dialog box using F12.
 
**Actual Result:-**
Proper role is not defined for the 'New Project' ,'Import File' and 'Untitled' elements showing under Project dialog box.

**Expected Result:-**
Proper role like button or link should be defined for the 'New Project' ,'Import File' and 'Untitled' elements showing under Project dialog box.

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Suggested fix:-**
 1- Using aria-label to give an invisible name to element, where name/title property is not used  
e.g.  
'Close' button can have only ‘X’, to make it more meaningful aria-label attribute is used with ‘Close’ text to help AT’s without showing the content on UI 
 
<div id="box"> 
   This is a pop-up box. 
    < button aria-label="Close" onclick="document.getElementById('box').style.display='none';" class="closebutton"> X     < /button>                                          
</div> 
 
2-   Using ARIA role to expose the control type of user interface component 
e.g. 
< div role="toolbar" tabindex="0" id="customToolbar" >  
< img src="img/btn1.gif" role="button" tabindex="-1" alt="Home" id="b1" title="Home" > 
 
3-   Using ARIA attributes to expose the element’s state  
e.g. 
< div role="button" id="customToolbar" aria-checked=”” aria-disabled=”” aria-haspopup=”” aria-expanded=”” aria-selected=”” >  
 
Reference: 
https://www.w3.org/TR/WCAG20-TECHS/aria.html 
https://www.w3.org/TR/WCAG20-TECHS/ARIA16.html  
https://www.w3.org/TR/WCAG20-TECHS/ARIA4.html  
https://www.w3.org/TR/WCAG20-TECHS/ARIA5.html 

**Please refer the attachment**
![mas4 2 1_projects](https://user-images.githubusercontent.com/30459830/28609891-090b4128-7204-11e7-918f-053dfcf5c5d8.jpg)

[MAS4.1.2_Projects.zip](https://github.com/Microsoft/pxt/files/1176066/MAS4.1.2_Projects.zip)

**Comment by v-amik** [comment]
review done

**Comment by veler** [comment]
@v-amik Issue fixed and available to test

**Comment by v-amik** [comment]
As per process, bug should be assigned to HCL for regression.

For Any external/Won’t Fix/By Design bug, there is a process to go through by Product team. 
Please go through the process here:

https://microsoft.sharepoint.com/teams/ceacc/SitePages/Exception process.aspx
https://microsoft.sharepoint.com/teams/ceacc/SitePages/External Bug Process.aspx

**Comment by v-mesala** [comment]
Bug is verified on URL :https://makecode.microbit.org/acc and bug is fixed .Hence closed the bug and added the Closed tag.
